server/routes/api/authenticationProviders/authenticationProviders.ts

```diff
@@ -1,26 +1,27 @@
 import Router from "koa-router";
 import { sequelize } from "@server/database/sequelize";
 import auth from "@server/middlewares/authentication";
+import validate from "@server/middlewares/validate";
 import { AuthenticationProvider, Event } from "@server/models";
 import { authorize } from "@server/policies";
 import {
   presentAuthenticationProvider,
   presentPolicies,
 } from "@server/presenters";
 import { APIContext } from "@server/types";
-import { assertUuid, assertPresent } from "@server/validation";
-import allAuthenticationProviders from "../auth/providers";
+import allAuthenticationProviders from "../../auth/providers";
+import * as T from "./schema";
 
 const router = new Router();
 
 router.post(
   "authenticationProviders.info",
   auth({ admin: true }),
-  async (ctx: APIContext) => {
+  validate(T.AuthenticationProvidersInfoSchema),
+  async (ctx: APIContext<T.AuthenticationProvidersInfoReq>) => {
     const { id } = ctx.request.body;
```

Comment:
In order to get the benefit of the typed response we need to change the handlers to use `ctx.input.body` instead of `ctx.request.body`, the former is typed correctly after the validate middleware.

I just realized this also applies to https://github.com/outline/outline/pull/4859where I missed it!